src/platform/packages/shared/kbn-esql-validation-autocomplete/src/definitions/generated/operators.ts

```diff
@@ -5474,8 +5122,6 @@ export const operatorFunctionDefinitions = [
   notInDefinition,
   notLikeDefinition,
   notRlikeDefinition,
-  isNullDefinition,
```

Comment:
If we remove them, where are they defined? If I remember we had them somewhere handwritten and we removed them because they added them in their definitions. Check this PR the all_opeators file https://github.com/elastic/kibana/pull/216422

Reply:
Thanks @stratoula I know we need to add these back before this merges. That's why this is marked "Do not merge" :)

Reply:
(Added in https://github.com/elastic/kibana/pull/219854/commits/743fb037f42145d9315234b6e46e0102b6cd2a1c)

Reply:
I see, a comment about it would help to understand 😉

Reply:
I marked the PR as blocked so I didn't expect a review here except with respect to making sure https://github.com/elastic/kibana/pull/219779 was working correctly.

Still, I can tell that my messaging could have been clearer :)